.github/workflows/code-checks.yml

```diff
@@ -32,17 +35,18 @@ jobs:
           ${{ runner.os }}-pip-
 
     - name: Install dependencies
-      env:  # TODO: drop this when we will be using regular releases for testing
-        PACKAGE_NAME: pytorch
       run: |
-        pip install mypy==0.982
+        pip install mypy==0.982  # TODO: pull this verion deom requirements/test.txt
```

Comment:
This TODO is not correct, mypy is not a requirement of this project. Just only used here in CI